src/main/java/adris/altoclef/AltoClefCommands.java

```diff
@@ -180,36 +251,41 @@ public static void TEMP_TEST_FUNCTION(AltoClef mod, String arg) {
                 break;
             case "avoid":
                 // Test block break predicate
-                mod.getConfigState().avoidBlockBreaking((BlockPos b) -> (-1000 < b.getX() && b.getX() < 1000)
-                        && (-1000 < b.getY() && b.getY() < 1000)
-                        && (-1000 < b.getZ() && b.getZ() < 1000));
+                mod.getConfigState()
+                   .avoidBlockBreaking(pos -> (-1000 < pos.getX() && pos.getX() < 1000) &&
+                                              (-1000 < pos.getY() && pos.getY() < 1000) &&
+                                              (-1000 < pos.getZ() && pos.getZ() < 1000));
                 Debug.logMessage("Testing avoid from -1000, -1000, -1000 to 1000, 1000, 1000");
                 break;
             case "portal":
                 mod.runUserTask(new EnterNetherPortalTask(new ConstructNetherPortalBucketTask(), Dimension.NETHER));
                 break;
             case "kill":
-                List<ZombieEntity> zombs = mod.getEntityTracker().getTrackedEntities(ZombieEntity.class);
-                if (zombs.size() == 0) {
-                    Debug.logWarning("No zombs found.");
+                List<ZombieEntity> zombies = mod.getEntityTracker().getTrackedEntities(ZombieEntity.class);
+                if (zombies.isEmpty()) {
+                    Debug.logWarning("No zombies found.");
                 } else {
-                    LivingEntity entity = zombs.get(0);
+                    LivingEntity entity = zombies.get(0);
                     mod.runUserTask(new KillEntityTask(entity));
                 }
                 break;
             case "craft":
                 // Test de-equip
-                new Thread() {
+                new Thread() { // I just *love* creating new threads like this
```

Comment:
when java isn't ur main language, that's what happens lol

Reply:
yeah. You should really use a thread executor and scheduler for this kind of stuff, then submit lambdas to it (or classes extending a functional interface if it's longer than like 15 lines.)